snp_blending/urdf/mh180_120_macro.xacro

```diff
@@ -0,0 +1,181 @@
+<?xml version="1.0"?>
```

Comment:
Add license/notice file(s) since we lifted this xacro from the motoman repository. See [here](https://github.com/ros-industrial/reach/tree/master/reach_ros/demo/model/motoman_sia20d) for a similar reference